next.config.js

```diff
@@ -1,4 +1,10 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {};
+module.exports = nextConfig;
+// const path = require(`path`);
 
-module.exports = nextConfig
+// module.exports = {
+//   sassOptions: {
+//     includePaths: [path.join(__dirname, `styles`)],
+//   },
+// };
```

Comment:
What is the purpose of this?

Reply:
https://nextjs.org/docs/app/building-your-application/styling/sass

Reply:
Why is it commented out? Are you gonna use it, if so, uncomment it, otherwise delete it.

Reply:
deleted